### ISSUE TITLE
Ekstra solar pannel option

### ISSUE THREAD

**Issue by perjmolsen** [reptilex/tesla-style-solar-power-card#153]
Hi

First of all - it a super cool card you have made.
I have two solar inverters - it is possible that you can make an option to add an extra inverter?

Kind regards
Per

**Comment by tomcoleman** [comment]
that would be cool! what dual inverter setup do you have?

**Comment by perjmolsen** [comment]
My setup is a  Growatt SPH3600 inverter with 6.5kwh battery and 18 solar pannels.
I just got an extra Growatt MIC3000tl-x added to my setup.
Because when the battery is full the sph3600 is only using the 3600W of the total 6600W from my solar panels - 
When adding the MIC3000 on the 8 of the 18 pannels then I got the full solar power.

Then it would be cool to see both my inverters in the tesla power card :)
